src/main/resources/templates/db_scripts/schema.json

```diff
@@ -114,6 +115,37 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "item.identifiers",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        },
+        {
+          "fieldName": "requesterId",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
+      ],
+      "fullTextIndex": [
+        {
+          "fieldName": "item.identifiers",
+          "arraySubfield" : "value",
+          "arrayModifiers": ["identifierTypeId"]
+        },
+        {
+          "fieldName": "item.barcode",
+          "tOps": "ADD"
+        },
+        {
+          "fieldName": "item.title",
+          "tOps": "ADD"
+        },
+        {
+          "fieldName": "requester.barcode",
+          "tOps": "ADD"
```

Comment:
Why have indexes been added for properties other than the item identifiers?

Reply:
These properties are used in request search box, actually the search query is following: 
`((requesterId=="900" or requester.barcode="900*" or item.title="900*" or item.barcode="900*" or itemId=="900")) sortby requestDate`

Reply:
Hmm, I'm not sure we should add indexes in unrelated pull requests only because they are used in the search.

@skoczko @julianladisch @adamdickmeiss What do you think, should we back fill indexes for these properties?

Reply:
You're right. Should I fill in a bug to create indexes?

Reply:
How do we know that it is a bug?

Reply:
Maybe not, however RMB logs a warning that full text search are being performed without index.

Reply:
@skoczko @julianladisch @adamdickmeiss Is this considered a bug?

Reply:
@bohdan I think it is worth asking @julianladisch about the upcoming normalisation changes for identifiers in inventory, if those require changes to indexes, then it might not be worth adding any here only to have to change them soon.

Reply:
@julianladisch responded that the normalisation work will require index re-create (see RMB-499).

Because of this, I've removed all the previously added indexes. Please note, that CQL search query may be different with/without index, depending on which approach we will chose on UI for search:
* If we allow user to limit search for particular identifier type (e.g. search by ISBN only, as we do in inventory) - then `/@` is not supported without index. With index the query will be `item.identifiers = /@identifierTypeId = 'id' 'value'` but without index this feature is not supported and have to be something like `item.identifiers = 'id' and item.identifiers = 'value'`
* In case the search will not require limitation by identifier type then the CQL will be the same.

Reply:
Hmm, maybe I've approved this too soon.

@bohdan-suprun Please can you follow up with @julianladisch about how the query will work in inventory when the value has been normalised.

Ideally, the client wouldn't have to change it's query between these changes.

If they will have to, we might want to hold off merging this and apply the normalization within the same pull request.

Reply:
I don't think that the normalization will affect CQL query. My main point that the CQL query will be different when we use an index (either normalized or not) or don't.

This depends on which search approach we will chose on UI:
* If we chose the approach when we have a drop-down to pick an identifier type, like what we currently have in inventory:
![image](https://user-images.githubusercontent.com/9094650/73280190-dcc6d580-41f6-11ea-825e-6054b8eea132.png)
then the CQL will be different with and without index.
* If the search box is a simple keyword-like search (what we have currently in requests, when we search is `requesterId = '...' or item.barcode = '...' or etc...`) then CQL remains the same for both cases.

Reply:
@bohdan-suprun My understanding is that the normalization will work by interpreting some CQL queries differently.

I think that means we need to understand what kind of queries will need to be used after the normalization changes are made.

The query (or queries) and indexes should match those in inventory, after the normalization changes.

Does that make sense?

Reply:
I've posted a question in RMB-499.

Reply:
My suggestion is to merge this pull request and not to wait for RMB-499.
RMB-499 will be released in RMB v29.3.x.
mod-circulation-storage uses RMB v26.2.3 and requires an upgrade: https://issues.folio.org/browse/CIRCSTORE-183
Using the barcode normalization from RMB-499 requires a new index. And it probably requires changes to CQL to use the (virtual) normalized field.

Reply:
Agreed, this should be merged to allow for the work to populate these properties to be done.

A new issue needs to be raised for supporting the appropriate queries when the normalisation work is ready. @bohdan-suprun please can you raise that?

Reply:
The https://issues.folio.org/browse/CIRCSTORE-186 has been created.